### ISSUE TITLE
Update the game to version 2.0.9

### ISSUE THREAD

**Issue by user1-github** [flathub/net.redeclipse.RedEclipse#15]
Hi,

For a bit of context, the current version 2.0.0 was in many ways a massive downgrade compared to the previous version 1.6.0 (like much fewer maps, among other things). Version 2.0.0 has also been available on Steam for a few years now, but earlier this year it was upgraded to 2.0.9 beta for everyone on Steam.

It's still unknown when the full version 2.1.0 is going to be released, so that's why many are excited to play 2.0.9 beta. Unfortunately, Steam is currently the only easy way to play the new version and I've seen folks trying to find other ways to play it.

That's why I'd like to ask if it's feasible to update the game to version 2.0.9 beta? (even if technically the latest stable version is still considered 2.0.0). Currently there's no source tarball for version 2.0.9, so if you need it to build the new version, I can ask the devs on discord if they can release one.

**Comment by AsciiWolf** [comment]
Sure, I can make a "beta" branch in this repo and if you want, feel free to make a Pull Request with the 2.0.9 beta version to this branch. :-) Let me know if this is ok with you. I sadly do not have enough time to be able to work on this myself at the moment.

**Comment by user1-github** [comment]
> Sure, I can make a "beta" branch in this repo and if you want, feel free to make a Pull Request with the 2.0.9 beta version to this branch. :-) Let me know if this is ok with you. I sadly do not have enough time to be able to work on this myself at the moment.

Ok, thank you. So I understand it's against Flathub's policy to release software version that's considered "beta" to the main branch?

For reference, the Steam 2.0.9 beta release is released for everyone (not just those who are on the Steam beta branch).

I'm not sure if creating a separate beta Flathub branch for 2.0.9 makes a lot of sense because most regular users will still only see 2.0.0 and will most likely not know about a separate beta branch with the new version.

**Comment by AsciiWolf** [comment]
> Ok, thank you. So I understand it's against Flathub's policy to release software version that's considered "beta" to the main branch?

Yes, it is. The only exception are applications that have very outdated and broken stable release and the only actively maintained releases are in beta.

**Comment by user1-github** [comment]
> Yes, it is. The only exception are applications that have very outdated and broken stable release and the only actively maintained releases are in beta.

Thanks, understandable. Then I guess it's better to wait for the full 2.1.0 release.

> Are you sure that the 2.0.9 Steam beta files are directly in the Flatpak and not fetched by the Steam client (updater) itself?

I didn't really understand the question, but I see that the Red Eclipse Flatpak relies on the source code from the release tarballs available [here](https://github.com/redeclipse/base/tags). As you can see, 2.0.0 is the latest one. I honestly don't know how they made the 2.0.9 beta release for Steam.

**Comment by AsciiWolf** [comment]
They are probably compiling it directly from Git. Yeah, it will be better to wait for the actual 2.1.0 release.

**Comment by AsciiWolf** [comment]
Yes, you can make another Flatpak with a different namespace. For example `net.redeclipse.RedEclipse-Legacy` or something similar.

**Comment by user1-github** [comment]
Awesome, thanks!

**Comment by user1-github** [comment]
Test builds of 2.0.9/2.1.0 can now be tried at #27
